mreg/api/v1/zonefile.py

```diff
@@ -64,20 +67,41 @@ def generate(self):
             data += srv.zf_string(zone.name)
         return data
 
+def get_ipaddresses(network):
+    from_ip = str(network.network_address)
+    to_ip = str(network.broadcast_address)
+    where_str = "ipaddress BETWEEN '{}' AND '{}'".format(from_ip, to_ip)
```

Comment:
https://docs.djangoproject.com/en/dev/ref/models/querysets/#range

Reply:
Kan teste den... minnes det var noe brukne saker, men det var kanskje "__lt" og "__gt".

Reply:
Range seems to be ok.